src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -10,6 +10,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 
+#pragma warning disable SA1011

```

Comment:
I know we've discussed about this but do you mind adding a comment above explaining briefly why we need this, as well as maybe a link to a new issue for removing it once we move to new analyzers

Reply:
Context: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2927